cli/QuickstepCli.cpp

```diff
@@ -150,6 +154,14 @@ DEFINE_bool(initialize_db, false, "If true, initialize a database.");
 DEFINE_bool(print_query, false,
             "Print each input query statement. This is useful when running a "
             "large number of queries in a batch.");
+DEFINE_string(profile_file_name, "",
```

Comment:
Thanks @hbdeshmukh for those suggestions. I'm not sure I completely agree with them, but before we discuss them, are you suggesting that we make these changes in the current PR or put it off to another one? If the latter, let's merge and discuss this on another thread (maybe a JIRA issue?).